external/curl/Kconfig

```diff
@@ -0,0 +1,186 @@
+menu "CURL Options"
+
+config ENABLE_CURL
+	bool "Enable CURL"
+	default y
+	---help---
+		Libcurl supports various Application level protocols.
+		HTTP, FTP, Gopher etc.
+
+if ENABLE_CURL
+config DISABLE_HTTP
+	bool "Disable HTTP"
+	default n
+
+if !DISABLE_HTTP
+config USE_NGHTTP2
+	bool "Use NGHTTP2"
+	default n
+
+config USE_ZLIB
+	bool "Use Zlib Compression"
+	default n
+
+config DISABLE_COOKIES
+	bool "Disable COOKIES"
+	default n
+endif #DISABLE_HTTP
+
+config DISABLE_DICT
```

Comment:
I think that DISABLE_*** makes developers confused.
Because developers get used to selecting ENABLE_***
Can you change it?

Reply:
Actually CURL follows the convention "Enable everything by default and disable unwanted features" which is different from Tizen RT Config convention. I am afraid changing the CURL convention might lead to instability.